helm-charts/aws-auth/bitops.config.yaml

```diff
@@ -4,7 +4,7 @@ helm:
     timeout: 200s
     debug: true
     atomic: false
-    force: false
+    force: true
```

Comment:
this seems like a fairly significant change, what is the impact?

Reply:
This is to ensure that any change made to the aws-auth chart get's deployed correctly.